src/main/java/core/basesyntax/BinaryString.java

```diff
@@ -7,6 +7,11 @@
      * а возвращает String с представлением этого числа в двоичном виде.
      */
     public String toBinaryString(int value) {
-        return null;
+        StringBuilder binary = new StringBuilder("");
```

Comment:
awesome 👍 
but let's rename variable `binary`. It can be adjective

Reply:
done